core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -187,7 +187,7 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
   override def create(user: Identity, entityName: FullyQualifiedEntityName)(implicit transid: TransactionId) = {
     parameter('overwrite ? false) { overwrite =>
       entity(as[WhiskActionPut]) { content =>
-        val request = content.resolve(user.namespace)
+        val request = content.resolve(user.namespace.name)
```

Comment:
should we instead change these method signature to accept the new Namespace type instead?

Reply:
Addressed in latest commit